lib/devices_list/devices_list_view.dart

```diff
@@ -106,22 +97,58 @@ class DeviceListScreenState extends State<DevicesListScreen> {
     Fimber.d("reassemble");
     super.reassemble();
   }
+}
+
+class _HexPainter extends CustomPainter {
+  const _HexPainter({
+    this.backgroundColor = Colors.white,
+    this.foregroundColor = Colors.black,
+  }) : super();
 
+  final Color foregroundColor;
```

Comment:
It looks like it can't be private as long as it's optional. I'm getting this error:

![Zrzut ekranu 2019-03-21 o 11 41 51](https://user-images.githubusercontent.com/1166994/54747177-746a9900-4bce-11e9-9059-2ec714a0aedc.png)

Explanation is here: https://github.com/dart-lang/sdk/blob/b0855ffd35ad4878fb3b36d8f5ff00a421719e7c/docs/language/dartLangSpec.tex#L1427

Look at the implementation of `CircleAvatar`. They're doing it the same way: `backgroundColor` and `foregroundColor` fields are not private. So I think it's all right.